website_snippet_carousel_product/static/src/js/s_product_carousel_frontend.js

```diff
@@ -3,24 +3,25 @@
 odoo.define("website_snippet_carousel_product.s_product_carousel", function(require) {
     "use strict";
 
-    var core = require("web.core");
-    var sAnimation = require("website.content.snippets.animation");
+    const core = require("web.core");
+    const sAnimation = require("website.content.snippets.animation");
 
-    var _t = core._t;
+    const _t = core._t;
 
     sAnimation.registry.js_product_carousel = sAnimation.Class.extend({
         selector: ".js_product_carousel",
+        disabledInEditableMode: false,
```

Comment:
Why?

Reply:
It is needed for see the preview of the snippet... If we don't use this flag it is initialized as true and we couldn't see the preview.

Reply:
Thanks, @Tardo :+1: